docs/references/t-ecdsa-how-it-works.mdx

```diff
@@ -1,73 +1,110 @@
 
-The derivation of further ECDSA keys from a canister root key can be done without involvement of ICP as well to facilitate certain use cases.
+The derivation of further t-sig keys from a canister root key can be done without involvement of ICP as well to facilitate certain use cases.
 
 ![Threshold ECDSA Key derivation](./_attachments/key_derivation.png)
 
-Threshold ECDSA master keys are always referred to through **key identifiers** in the threshold ECDSA API (as well as in the NNS proposals for managing the rollout). The key identifiers comprise an elliptic curve name and an identifier, e.g., an example key identifier is the 2-tuple `(secp256k1, test_key_1)`. Those key identifiers are used by the system to refer to the correct key, e.g., for selecting the key share when computing a signature or in the implementation of the XNet routing of API calls and responses to/from the ECDSA-enabled subnet holding the key with the corresponding identifier.
+Threshold master keys are always referred to through **key identifiers** in the t-sig APIs (as well as in the NNS proposals for managing the rollout). The key identifiers comprise an elliptic curve name and an identifier, e.g., an example key identifier is the 2-tuple `(secp256k1, test_key_1)` for ECDSA or `(bip340secp256k1, test_key_1)` for Schnorr. Those key identifiers are used by the system to refer to the correct key, e.g., for selecting the key share when computing a signature or in the implementation of the XNet routing of API calls and responses to/from the t-sig-enabled subnet holding the key with the corresponding identifier.
 
-There are currently two master keys deployed, a test and a production key.
+There are currently three master keys deployed, an ECDSA test key, an ECDSA production key, and a Schnorr test key.
 
-- `(secp256k1, test_key_1)`: the test key deployed on a single 13-node subnet.
-- `(secp256k1, key_1)` the production key deployed on two high-replication subnets, one activated for signing, and the other one for backing up the key for better key availability.
+- `(secp256k1, test_key_1)`: the ECDSA test key deployed on a single 13-node subnet.
+- `(secp256k1, key_1)` the ECDSA production key deployed on two high-replication subnets, one activated for signing, and the other one for backing up the key for better key availability.
+- `(bip340secp256k1, test_key_1)`: the Schnorr test key deployed on a single 13-node subnet.
```

Comment:
```suggestion
- `(secp256k1, test_key_1)`: The ECDSA test key deployed on a single 13-node subnet.
- `(secp256k1, key_1)`: The ECDSA production key deployed on two high-replication subnets, one activated for signing, and the other one for backing up the key for better key availability.
- `(bip340secp256k1, test_key_1)`: The Schnorr test key deployed on a single 13-node subnet.
```

Reply:
Is there a production key for Schnorr?

Reply:
Not yet, but likely in a few weeks after testing